src/index.d.ts

```diff
@@ -7,7 +7,7 @@ export interface Location {
  * An object representing an interaction with one or more registered geofence radii.
  */
 export interface GeofenceEvent {
-  id: string
+  geofenceId: string
```

Comment:
changed this name to match ios makes more sense since the events themselves dont have ids, unfortunately android 4.0.0 already released with `GeofenceEvent.id` property but prolly not huge deal